docs/core-concepts/working-with-templates/data-objects.md

```diff
@@ -0,0 +1,47 @@
+# Data Available in Templates
+
+Apostrophe makes different data available for you in templates depending on the context. Most of this is covered in other sections as well, but this page serves to collect that information in one place, as well as highlight options not covered elsewhere.
+
+::: tip
+Anything attached to the `req.data` object becomes visible as the `data` object in Nunjucks templates when rendering pages and other templates. Before a page is rendered, you can add properties onto `req.data` to make any new information that you need available in a template.
+:::
+
+## `data.global`
+
+`data.global` is available everywhere and always represents the same `apostrophe-global` document with the same information everywhere. Read more about this on the [global doc reference page](/core-concepts/global-settings/global.html#the-global-doc-sharing-content-across-pages).
+
+## `data.page`
+
+The `data.page` object is available in any page, piece "show page" (e.g., `show.html` -- these aren't "pages" in the sense that they might extend `apostrophe-pages`), and widget template. It contains information about the current page's document (e.g., title, areas). There is a bit more to think about if  you are using it on show page or widget templates, but most of the time you will probably be using this directly on a normal page or index page template.
+
+In `show.html` templates it references the index page that the show "page" belongs to at a given URL (if you have one index page for the piece type this is simple, with multiple index pages for a type it's slightly more complicated).
```

Comment:
I wouldn't say that's complicated, I'd just say it depends on which index page (which "blog") you are looking at.